src/main/config/openmrs-runtime.properties

```diff
@@ -10,6 +10,6 @@ connection.password=test
 tomcatport=8081
 application_data_directory=appdata
 reset_connection_password=true
-vm_arguments=-Xmx512m -Xms512m -XX:NewSize=128m --add-exports=java.desktop/com.apple.eawt=ALL-UNNAMED
+vm_arguments=-Xmx512m -Xms512m -XX:NewSize=128m --add-exports=java.desktop/com.apple.eawt=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED
```

Comment:
Is this a part of github actions?

Reply:
not really let me remove it and add it in another PR